packages/BetterPhpDocParser/PhpDocParser/StaticDoctrineAnnotationParser/ArrayParser.php

```diff
@@ -96,6 +96,13 @@ private function resolveArrayItem(BetterTokenIterator $tokenIterator): array
             $tokenIterator->next();
         }
 
+        $tokenIterator->tryConsumeTokenType(Lexer::TOKEN_PHPDOC_EOL);
+
+        if ($tokenIterator->isCurrentTokenTypes([Lexer::TOKEN_CLOSE_CURLY_BRACKET, Lexer::TOKEN_COMMA])) {
+            // it's a value, not a key
+            return [null, $key];
+        }
+
```

Comment:
I often struggle with the right tokens to consume. This is amazing work :+1:  Thank you